build/archive.go

```diff
@@ -38,7 +38,7 @@ func findNodePkgs(path string) []string {
 
 // Build does nothing for VendoredArchives
 func (builder *VendoredArchiveBuilder) Build(m module.Module, force bool) error {
-	return errors.New("VendoredArchive packages are expected to be pre-built")
+	return fmt.Errorf("VendoredArchive packages are expected to be pre-built")
```

Comment:
You don't need to use `fmt.Errorf` when you're not using a format string. Also, you should try to use the least functionality you require for a task (this helps communicate intent). This would be an appropriate place ot use `errors.New`.

The same goes for the other files in `build/`.